client/blocks/comment-detail/index.jsx

```diff
@@ -68,8 +69,10 @@ export class CommentDetail extends Component {
 	}
 
 	deleteCommentPermanently = () => {
-		const { commentId, deleteCommentPermanently } = this.props;
-		deleteCommentPermanently( commentId );
+		const { commentId, deleteCommentPermanently, translate } = this.props;
+		if ( window.confirm( translate( 'Delete this comment permanently?' ) ) ) {
```

Comment:
Do you mean something like `if ( ! window || window.confirm() )` or do we have a pattern going on (tried searching, seems nobody else ever checked for window on confirm)?